src/client/pps.go

```diff
@@ -46,6 +46,12 @@ const (
 	// PPSHostPathVolume is the name of the volume that uses the
 	// aforementioned hostpath.
 	PPSHostPathVolume = "pachyderm-worker"
+	// PPSWorkerUserContainerName is the name of the container that runs
+	// the user code to process data.
+	PPSWorkerUserContainerName = "user"
+	// PPSWorkerSidecarContainerName is the name of the sidecar container
+	// that runs alongside of each worker container.
+	PPSWorkerSidecarContainerName = "sidecar"
```

Comment:
My intention was to distinguish between `pachd` and `sidecar`.  To be, `pachd` is the centralized servers that store a lot of state, handle user requests, store cache, etc.  `sidecar` is, well, a sidecar to the user container.  I feel like having the user see a bunch of `pachd` containers running inside workers can be a little confusing.